src/Compilers/Core/Portable/Symbols/ISymbolExtensions.cs

```diff
@@ -100,5 +93,24 @@ internal static INamespaceSymbol GetNestedNamespace(this INamespaceSymbol contai
 
             return null;
         }
+
+        /// <summary>
+        /// Returns whether this type inherits from the given WellKnownType
+        /// </summary>
+        internal static bool InheritsSpecialType(this ITypeSymbol originalType, SpecialType specialType)
```

Comment:
>InheritsSpecialType [](start = 29, length = 19)

It doesn't look like there is a real need in this method.

Reply:
Removed.